tfrecords/create_tfrecords.py

```diff
@@ -41,51 +40,56 @@ def load_sidewalks_dataset(args):
     return train_ds, val_ds
 
 
-def normalize_img(image: np.ndarray, label: np.ndarray) -> Tuple[tf.Tensor, tf.Tensor]:
-    image = tf.cast(image, tf.float32) / 255.0
-    return image, label
```

Comment:
Why is this deleted?

Reply:
since I am going to apply `mobilenet_v2.preprocess_input` to the input dataset, I guess it does perform normalization

Reply:
Then let's apply that here during the creation of TFRecords?

Reply:
Possible.

But just wanted to let readers know it is possible to do so in Transform since this project focuses on ML pipeline not the dataset itself. WDYT?

Reply:
Makes perfect sense to me.